### ISSUE TITLE
setPrecision function

### ISSUE THREAD

**Issue by hhls** [uklimaschewski/EvalEx#287]
How can I keep exactly a few decimal places like using BigDecimal.setScale(outScale, outMode)
setPrecision(num) seems to only decide how many digits

**Comment by RobertZenz** [comment]
Yes, that would calling `setScale` on the result. Or do you want to perform all operations in the expression with that scale?

**Comment by hhls** [comment]
> Yes, that would calling `setScale` on the result. Or do you want to perform all operations in the expression with that scale?

I misunderstood precision and how many decimal places to keep, you are correct👍🏻
